specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -8045,8 +8048,11 @@
         },
         "headers": {
           "type": "object",
-          "x-ms-format": "dfe-key-value-pairs",
-          "description": "Represents the headers that will be sent to the request. For example, to set the language and type on a request: \"headers\" : { \"Accept-Language\": \"en-us\", \"Content-Type\": \"application/json\" }. Type: dictionary (or Expression with resultType dictionary)."
+          "description": "Represents the headers that will be sent to the request. For example, to set the language and type on a request: \"headers\" : { \"Accept-Language\": \"en-us\", \"Content-Type\": \"application/json\" }. Type: string (or Expression with resultType string).",
+          "additionalProperties": {
+            "type": "string",
+            "x-ms-format": "dfe-string"
```

Comment:
If you use x-ms-format, then its type must be object.

Reply:
According to our test, if we define the type as object. For additionalProperties the generated model is `public IDictionary<string, BinaryData> Headers { get; }` and it's not our expected.